release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1859,9 +1846,9 @@ In this update, `systemd` service only sets the default RPS mask for virtual int
 
 * Previously, the `openshift-router` process briefly ignored the `SIGTERM` shutdown signal. This caused containers to ignore a Kubernetes shutdown request, resulting in one hour shutdown times. With this update, the router now responds to `SIGTERM` signals. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076297#[*BZ#2076297*])
 
-[discrete]
-[id="ocp-4-11-samples-bug-fixes"]
-==== Samples
+* Previously, when an Ingress Controller for an admitted route was deleted or a sharding configuration was added, a false status of `admitted` was given. With this update, the Ingress Controller clears the status of an `unadmitted` route, avoiding the false status scenario. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1944851[*BZ#1944851*])
+
+* Previously, {product-title} clusters installed using version 4.7 or earlier maintained a service.beta.kubernetes.io/aws-load-balancer-internal annotation value of 0.0.0.0/0. Clusters that were installed using 4.8 or later have the annotation value true. AWS cloud-provider implementations that check for the annotation value true would return the wrong result if the value was 0.0.0.0/0. As a result, cluster upgrades to 4.10 would not complete. With this update, the annotation value is normalized to true so that the cluster upgrades can complete. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055470[*BZ#2055470*])
```

Comment:
In my suggested text for this line, I included back-ticks but I think they were rendered in the comment rather than being printed explicitly. As such, a copy and paste from that comment might have not captured the back-ticks. Please add back-ticks for each instance of the following elements in this line:

* `service.beta.kubernetes.io/aws-load-balancer-internal`
* `0.0.0.0/0`
* `true`